src/inverseRobot.js

```diff
@@ -4,10 +4,24 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const invRobot = {};
+
+  for (const key in robot) {
+    if (Object.hasOwnProperty.call(robot, key)) {
+      const value = robot[key];
```

Comment:
you could simplify a bit with using `Object.entries()` to iterate only the own properties

```suggestion
for (const [key, value] of Object.entries(robot)) {
```